content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -114,11 +114,27 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
     curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< param "fullversion" >}}/g' | kubectl apply -f -
     kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
     ```
-
     {{< note >}}
     If you're using host-gateway use https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-host-gw.yml instead
     {{< /note >}}
 
+    {{< note >}}
+    If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line:
+    
+    ```powershell
+    wins cli process run --path /k/flannel/setup.exe --args "--mode=overlay --interface=Ethernet"
+    ```
+    in the flannel-host-gw.yml or flannel-overlay.yml file and specify your interface accordingly.
+    {{< /note >}}
+    
+    ```bash
+    # Example
+    curl -Lo flannel-overlay.yml https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
+    sed -i "s|=Ethernet|Ethernet0 2|g"
```

Comment:
I think this should be `"s|=Ethernet|=Ethernet0 2|g" flannel-overlay.yml`

You could also do
```
curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml | \
  sed "s|=Ethernet|=Ethernet0 2|g" | \
  kubectl apply -f -
```

to just do it without writing a file to disk

Reply:
Ops yes.
I was thinking to keep the step easy, that is:
- download the file
- have the user be sure of the content to check
- perform the change
- apply it
For me it was useful to check the content of the yml to understand what was wrong.
However, I have updated it to be consistent with the previous step.